src/content/1.7/project/maintainers-guide/summary-github-lifecycle.md

```diff
@@ -56,6 +56,84 @@ We sometimes receive support requests on GitHub, ranging from questions about th
 
 There are some Issues which do not fit in the previous categories, such as [releases Issues](https://github.com/PrestaShop/PrestaShop/issues/20804). They serve a specific purpose.
 
+## Projects
+
+Available in the ‘[Projects](https://github.com/PrestaShop/PrestaShop/projects)’ tab of the PrestaShop’s GitHub repository, versions kanbans organize the Core team’s work. Both the product and the tech teams use it to prepare and plan the next developments and, since we deal with an open source project, the purpose of having those kanbans public is to help the community and the external mainteners to know what is at stake.
+
+### Definition of states
+
+A ticket can either be an issue or a pull request. A pull request should always be linked to the issue it fixes. 
+That's said, there should only have issues in the kanban and it always have an owner, be it a PM or a developer.
+
+Community contributions are no longer part of the kanban. Only tickets developed or fixed by the Core team are included in the kanban, as it should reflect the Core team’s work. For example, an issue reports a bug prioritized for the 1.7.8.0, the ticket must appear in the 1.7.8 kanban while the linked pull request must have the 1.7.8.0 milestone.
```

Comment:
Is there a definition for Core Team ? Are there maintainers ?

Reply:
Yes, in the first part of the guide:

> Developers team a.k.a Core Team: this team takes care of the code and act as maintainer for the project.

Reply:
External (not in PrestaShop Corp) maintainer ?

Reply:
No, we are talking of the internal maintainers here (employees)

Reply:
Can you explicit that ?

Reply:
Yes, I just add this in the first part of the guide: https://github.com/PrestaShop/docs/pull/807/commits/103aa01248de15cb48f0b0c73bcf6f4feb944eed

Reply:
:heart_eyes: